lib/index.js

```diff
@@ -204,26 +227,27 @@ Validator.prototype._addFieldProperties = function (fieldName, field) {
   this.model.setEach(fieldName, data);
 };
 
-Validator.prototype._validate = function (fieldName) {
+Validator.prototype._validate = function (fieldName, callback) {
   this.model.del(fieldName + '.messages');
+  this.model.set(fieldName + '.validating', true);
+  var serial = this.model.increment(fieldName + '.serial');
+  var self = this;
 
   var field = this.model.get(fieldName);
-  if(!field || !field.hasOwnProperty('validations')) return true;
+  if (!field || !field.hasOwnProperty('validations')) {
```

Comment:
You can short-write this line into:
`if((!field || !field.hasOwnProperty('validations)) && callback) return callback(true);`
Then, of course, the question is if this is more readable. A middle ground might be:
`if(!field || !field.hasOwnProperty('validations)) {
  if(callback) return callback(true);
}`
Anyway, just FYI.